code/modules/admin/chat_commands.dm

```diff
@@ -81,3 +81,13 @@ GLOBAL_LIST(round_end_notifiees)
 	LAZYINITLIST(GLOB.round_end_notifiees)
 	GLOB.round_end_notifiees[sender] = TRUE
 	return "I will notify [sender] when the round ends."
+
+/datum/server_tools_command/reload_admins
+	name = "reload_admins"
+	help_text = "Forces the server to reload admins."
+	admin_only = TRUE
+
+/datum/server_tools_command/reload_admins/Run(sender, params)
+	load_admins()
+	log_admin("[sender] reloaded admins via IRC/discord command.")

```

Comment:
`via chat` (regrettably) more providers may be added in the future

Reply:
email